### ISSUE TITLE
Local filesystem support

### ISSUE THREAD

**Issue by jhjacobs81** [Clivern/Walrus#253]
It would be nice to have Walrus backup to the local filesystem for those of us that dont have S3 etc :)

**Comment by Clivern** [comment]
yeah it makes a lot of sense. not sure when but will let you know in this ticket.
